src/main/java/de/retest/recheck/report/TestReportFilter.java

```diff
@@ -130,6 +138,13 @@ static IdentifyingAttributesDifference filter( final Element element,
 						: new IdentifyingAttributesDifference( element.getIdentifyingAttributes(), diffs ) ) );
 	}
 
+	static InsertedDeletedElementDifference filter( final InsertedDeletedElementDifference insertedDeletedDiff,
+			final Filter filter ) {
+		final Element insertedDeleted =
+				insertedDeletedDiff.isInserted() ? insertedDeletedDiff.getActual() : insertedDeletedDiff.getExpected();
+		return filter.matches( insertedDeleted ) ? null : insertedDeletedDiff;
```

Comment:
Now we call `filter.matches(Element)` but I still think, that this needs to happen for general elements too.

Reply:
I believe it does work because `Filter#matches( Element, AttributeDifference )` defaults to `Filter#matches( Element )` if a filter only implements the latter. See the following assert: https://github.com/retest/recheck/blob/2e6fe13e88b514aae49331ab52f940bdea2ccd00/src/test/java/de/retest/recheck/report/TestReportFilterTest.java#L210